components/tag-list/tag-list-item-mixin.js

```diff
@@ -172,7 +172,7 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 			}
 
 			await this.updateComplete;
-
+			this.setAttribute('tabindex', '-1');
```

Comment:
It looks like the `tabindex` is managed [here in d2l-tag-list](https://github.com/BrightspaceUI/core/blob/main/components/tag-list/tag-list.js#L399). Does this new assignment cover a case that wasn't being handled?

Reply:
Yes, this handles the case where you focus the first tag and then press `Shift+Tab` to go backward. At that point focus is on the tag content, but because `tabindex` is `0` on the host, it focuses the host, but this handler doesn't run again (which would just trap it) I guess because it never lost focus in the light dom. Then you can move backward from there.